File: QuantConnect.AlphaStream/Requests/UnsubscribeRequest.cs
using QuantConnect.AlphaStream.Infrastructure;
using QuantConnect.AlphaStream.Models;
using RestSharp;

namespace QuantConnect.AlphaStream.Requests
{
    /// <summary>
    /// Unsubscribe to the Alpha given by the id in path. Pro-rate subscription billing for remainder of them billing cycle.
    /// </summary>
    [Endpoint(Method.POST, "alpha/{id}/unsubscribe")]
    public class UnsubscribeRequest : AttributeRequest<ApiResponse>
    {
        /// <summary>
        /// Identifier of the Alpha to unsubscribe
        /// </summary>
        [PathParameter("id")]
        public string Id { get; set; }
    }
}

File: AlphaStream/Requests/UnsubscribeRequest.py


class UnsubscribeRequest(object):
    """ Send subscription stop request for an Alpha """

    def __init__(self, alphaId):
        self.Id    = str(alphaId)
        self.Endpoint   = "alpha/" + self.Id + "/unsubscribe" 
        
    def GetPayload(self):
        payload = {
            "id" : self.Id
        }
        return payload